static/src/js/components/CollectionDetails/CollectionDetailsBody.js

```diff
@@ -57,6 +58,16 @@ const buildScienceKeywordList = (scienceKeywords) => {
   return scienceKeywordList
 }
 
+const buildNativeFormatList = (nativeFormats) => {
+  if (!nativeFormats.length) return []
```

Comment:
Shouldn't this return a React fragment or null? One condition returns markup, the other returns a javascript array.

Reply:
I just did the same thing that buildRelatedUrlsList and buildScienceKeywordList are doing

Reply:
ok -- well then those are wrong too! I think the 'issue' is that we always have a conditional that wraps the render when these are used so it never actually attempts to render the array. @trevorlang any idea how to best handle this? I know it hasn't caused issues (that I'm aware of) but definitely seems like a quick and easy improvement.